TECH_STACK.md

```diff
@@ -0,0 +1,93 @@
+# CAS Registration Project Technology Stack
+
+## Project Structure
+
+The project is divided into two main components:
+
+### Frontend (bciers/)
+The frontend application is built using modern web technologies and is organized as a monorepo using Nx.
+
+Directory structure:
+- `apps/` - Contains the main application code
+- `libs/` - Shared libraries and components
+- `.env` - Environment configuration
+- `docker-compose-*.yaml` - Docker compose configurations for different environments
+
+### Backend (bc_obps/)
+The backend is a Python-based application built with Django, featuring multiple services and modules.
+
+Directory structure:
+- `bc_obps/` - Core application code
+- `common/` - Shared utilities and components
+- `registration/` - Registration service
+- `reporting/` - Reporting service
+- `rls/` - Row Level Security implementation
+- `service/` - Main service layer
+
+## Technology Stack
+
+### Frontend Technologies
+- **Framework**: 
+  - Next.js (React-based framework)
+  - React
+
+- **Build & Development**:
+  - Nx (Monorepo management)
+  - Node.js
+  - Yarn (Package management)
+  - TypeScript
+  - Babel (JavaScript compiler)
+
+- **Testing & Quality**:
+  - Playwright (E2E testing)
+  - ESLint (Code linting)
+  - Prettier (Code formatting)
+  - Happo (Visual regression testing)
+  - Vitest (Unit testing)
+
+- **Styling**:
+  - TailwindCSS
+  - PostCSS
+
+### Backend Technologies
+- **Framework**:
```

Comment:
nit: It would also be great to mention that we are developing REST APIs with [Django-Ninja](https://django-ninja.dev/).